### ISSUE TITLE
Mention Jetpack Rewind at WordPress Updates page

### ISSUE THREAD

**Issue by simison** [Automattic/jetpack#8853]
![screen shot 2018-02-15 at 18 24 18](https://user-images.githubusercontent.com/87168/36267882-b5505ca6-127d-11e8-8705-5ae7a1ad76d0.png)

WordPress shows this message at **Updates** page:

> Important: before updating, please [back up your database and files](https://codex.wordpress.org/WordPress_Backups). For help with updates, visit the [Updating WordPress](https://codex.wordpress.org/Updating_WordPress) Codex page.

Since my backups are handled with Jetpack, should we mention it here somehow?

**Comment by jeherve** [comment]
We have a Backups JITM showing up there, but you may have dismissed those messages in the past maybe?

![screenshot 2018-02-15 at 22 04 38](https://user-images.githubusercontent.com/426388/36280974-4ef8972e-129c-11e8-84fc-cee41f3e0b76.png)

@withinboredom What are the conditions to display that JITM today? Does the content change when you can use Rewind?

**Comment by withinboredom** [comment]
The content is mostly agnostic, or specifically mentions Jetpack Backups in some places, if it's available. The jitm mentioned at the top will appear on the first wp-admin page view once JB takes over, IIRC. Summoning @dereksmart as I don't know much about it.

**Comment by withinboredom** [comment]
Ah, I misread the question! Yeah, we should do a jitm like that! 🤦‍♂️  ... my brain is a little fried this afternoon, sorry!

**Comment by simison** [comment]
@jeherve @withinboredom  yep so this is in a situation where I've enabled backups already.

Could we suppress or rather change the notification WP core gives here? Having _"please back up your database and files"_ -call to action might confuse customers. Replacing it with this would be neat:

> **Jetpack is handling your backups.** Use rewind if something goes wrong with the update. For help with updates, visit the Updating WordPress Codex page.

**Comment by simison** [comment]
I should add that this could encourage more people to hit that "update" button. Fear of breaking something when updating can be huge even if you know you have backups.

**Comment by beaulebens** [comment]
I think in the case where you've _already_ got Jetpack Backups (or VaultPress), I agree, we should suppress the core message, and maybe replace it with a "we've got your back" type message.

Possibly include information around when the last full backup was taken, offer the ability to take a new backup right now, or even better, just automatically trigger one _before_ the update takes place.
